### DISCUSSION TITLE
is_ipv4_address fix to check string of comma separated IP addresses instead of just one address.

### DISCUSSION THREAD

**Discussion by veso-k** [pfsensible/core#150] [Ideas]
Hello, all!
According tho the official documentation [here](https://galaxy.ansible.com/ui/repo/published/pfsensible/core/content/module/pfsense_dns_resolver)
`ip` is `required` string  and should contain `IPv4 or IPv6 comma-separated addresses to be returned for the host`. 

So when I have this structure it fails:
`item={'host': 'storage', 'domain': 'googleapis.com', 'ip': '192.168.0.1,192.168.0.2', 'descr': 'some description'}`
 and works with this:
 `item={'host': 'storage', 'domain': 'googleapis.com', 'ip': '192.168.0.1', 'descr': 'some description'}`

I want to create PR to change `is_ipv4_address` function in `core/plugins/module_utils/__impl/addresses.py` like this:

```
def is_ipv4_address(address):
    try:
        return all(
            isinstance(ip_address(addr.strip()), IPv4Address)
            for addr in address.split(",")
        )
    except ValueError:
        return False
```

Thank you!

**Comment by opoplawski** [comment]
This should be fixed with https://github.com/pfsensible/core/pull/154
